src/views/v2/components/loader/loader.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * View: Loader
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/components/loader/loader.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @version TBD
+ */
+
+if ( empty( $text ) ) {
+	$text = $this->get( 'text' ) ?: esc_html__( 'Loading...', 'event-tickets' );
+}
+
+if ( empty( $loader_classes ) ) {
+	$loader_classes = $this->get( 'classes' ) ?: [];
+}
```

Comment:
:no_entry_sign: Using short ternaries is not allowed

<sub>hash: 94b3b610f6f6d5aec42e15a75a3ab0d0</sub>